dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+| start   | `dmctl relay start --worker-name="worker1" source1`                                                 | --worker-name(default: "") | source-name            | start relay for a source on a worker.                                        |
+| stop    | `dmctl relay stop --worker-name="worker1" source1`                                                  | --worker-name(default: "") | source-name            | stop relay for a source on a worker.                                         |
+| purge   | `dmctl relay purge --sub-dir="2ae76434-f79f-11e8-bde2-024ac130008.000001" source1 mysql-bin.000006` | --sub-dir(default: "")     | source-name, file-name | purges relay log files of the DM-worker according to the specified filename. |
+
+### dmctl commands for DDL-LOCK
+
+| Command | Full Syntax Example                 | Flags | Arguments          | Description                                  |
+|---------|-------------------------------------|-------|--------------------|----------------------------------------------|
+| list    | `dmctl ddl-lock list task1`         |       | task-name          | show shard-ddl locks information for a task. |
+| unlock  | `dmctl ddl-lock unlock task1 lock1` |       | task-name, lock-id | force unlock un-resolved DDL locks.          |
+
+### dmctl commands for member
+
+| Command                    | Full Syntax Example                                   | Flags                                    | Arguments          | Description                                       |
+|----------------------------|-------------------------------------------------------|------------------------------------------|--------------------|---------------------------------------------------|
+| member list                | `dmctl member list --name="master-1" --role="master"` | --name(default: ""), --role(default: "") |                    | show members of current cluster by name and role. |
+| member offline             | `dmctl member offline --master-1`                     |                                          | master/worker-name | offline members of current cluster by name.       |
+| member evict-leader        | `dmctl member evict-leader master-1`                  |                                          | master-name        | evict leader for master node.                     |
+| member cancel-evict-leader | `dmctl member cancel-evict-leader master-1`           |                                          | master-name        | cancel evict leader for master node.              |
+
+### Optimised dmctl for interaction mode (optional)
+
+The current interaction of dmctl still has room for optimization, and I hope to take this opportunity to do some optimization of the interaction experience, focusing on the user can quickly select the command they want to enter through the keyboard, instead of entering commands by their memory.
+
+Here is a simple prototype demo:
+
+![dmctl](../media/new-dmctl.svg)
```

Comment:
PTAL at this demo @sunzhaoyang

Reply:
it's an optional work ( if i  have enough time) 😂